.github/workflows/test_api_node.yml

```diff
@@ -0,0 +1,43 @@
+name: Continuous Integration Node
```

Comment:
Let's align the workflow names with the tests folder name. It is either CI or API. Maybe even e2e?

Reply:
right